.github/workflows/reanimated-compatibility-check-nightly.yml

```diff
@@ -25,6 +25,8 @@ jobs:
             { version: '0.75', architecture: 'Fabric' },
             { version: '0.76', architecture: 'Paper' },
             { version: '0.76', architecture: 'Fabric' },
+            { version: '0.77', architecture: 'Paper' },
+            { version: '0.77', architecture: 'Fabric' },
```

Comment:
Do we want to remove checks for version 0.74 in order to align with our policy of supporting the last three versions?

Reply:
Sounds right

Reply:
Let's keep for support RN 0.74 for now since we're already dropping support RN 0.73 in Reanimated 3.17.